setup.md

```diff
@@ -0,0 +1,130 @@
+---
```

Comment:
Are we planning on reusing /getting-started/ for something? Unless there's a good reason to change, I really think we should keep the URLs consistent.